postBuild

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Last nightly build: No new nightly release
+NIGHTLY_TAG="nightly"
+NIGHTLY_REPO="galleon/scikit-decide"
+
+# Install scikit-decide from last nightly build if not yet installed
+skdecide_version=$(pip list | grep "scikit-decide")
```

Comment:
test below will crash if for any reason skdecide_version contains a newline character; add `| head -n 1` to make sure that it contains a single line